src/__tests__/interpreter-errors.ts

```diff
@@ -87,17 +87,17 @@ test('Error when deeply accessing property on undefined', () => {
   })
 })
 
-test('In case a function ever returns null, should throw an error as well', () => {
-  const code = `
-    const myNull = pair.constructor("return null;")();
-    myNull.prop;
-   `;
-  const context = mockContext(4)
-  const promise = runInContext(code, context, { scheduler: 'preemptive' })
-  return promise.then(obj => {
-    expect(obj).toMatchSnapshot()
-    expect(obj.status).toBe('error')
-    expect(context.errors).toMatchSnapshot()
-    expect(parseError(context.errors)).toBe('Line 3: Cannot read property prop of null')
-  })
-})
\ No newline at end of file
+// test('In case a function ever returns null, should throw an error as well', () => {
```

Comment:
Don't remove this test case. Just rewrite it to
```
null.prop;
```

`null` will be allowed from next semester onwards.

Reply:
@thomastanck i commented out the test case cause theres no way to create null in source (at least not yet). we can bring back the test once we introduce null?